addon/components/frost-list.js

```diff
@@ -155,6 +207,37 @@ export default Component.extend({
       })
     }
 
+    const componentKeyNamesForTypes = this.get('componentKeyNamesForTypes')
```

Comment:
Awesome :) Can we move this in 2 methods :) so we can test this :)

Reply:
Did you handle this?

Reply:
Just pushed that change.